fdbserver/storageserver.actor.cpp

```diff
@@ -1159,6 +1165,26 @@ struct StorageServer {
 		                      self->thisServerID,
 		                      SERVER_KNOBS->LATENCY_METRICS_LOGGING_INTERVAL,
 		                      SERVER_KNOBS->LATENCY_SAMPLE_SIZE),
+		    readKeyLatencySample("ReadKeyLatencyMetrics",
```

Comment:
I think this overall looks good, though I wonder if we should use a different name for this event. It may not be obvious that `ReadKey` refers to us resolving a key selector for a get key request

Reply:
Since this close to the name of the storage server API, I'm not sure what else to call it.

Reply:
It looks like the existing mapped range sample uses `GetMappedRangeMetrics`, maybe we could do `GetValueMetrics`, `GetRangeMetrics`, `GetKeyMetrics` that map more closely to the terms used in the API. Otherwise, perhaps something like `ReadKeyFromSelectorMetrics` would work.

Reply:
Sounds good to me.

Reply:
Although, "Metrics" is a bit general given that this is only latency.  For example, it may be useful to log other metrics around GetRange such as result size or record count, and it wouldn't be trivial to get those details into the same TraceEvent today, however I do think that is where they should go so I guess these names are fine.